src/kata6.joinNames.js

```diff
@@ -1,5 +1,10 @@
 const joinNames = (namesObj) => {
+    
+    const nameArray = namesObj.map(element => element.name);
+	const commaSeperatedNameString = nameArray.join(', ');
+	const lastCommaIndex = commaSeperatedNameString.lastIndexOf(',');
+    return commaSeperatedNameString.substr(0, lastCommaIndex) + ' &' + commaSeperatedNameString.substr(lastCommaIndex + 1);
```

Comment:
Please take time to indent your code - everything should be indented a tab inside a block so lines 4 and 5 here should be indented one tab further.